### PR TITLE
Update names of SUs in generated invoice

### PR DESCRIPTION
**Pull request by knikolla**
Partial https://github.com/nerc-project/coldfront-plugin-cloud/issues/116

### CODE REVIEW

File: src/openstack_billing_db/billing.py [hunk 1]
```diff
@@ -37,42 +43,22 @@ class ProjectInvoice(object):
...
-        return self.rates.gpu_k80
-
     @property
     def gpu_k80_su_cost(self):
         return self.rates.gpu_k80 * self.gpu_k80_su_hours
 
-    @property
-    def gpu_a2_su_rate(self):
-        return self.rates.gpu_a2
-
```
Comment (naved001) [inline src/openstack_billing_db/billing.py:75]: why did you delete these in this PR?

Comment (knikolla) [reply src/openstack_billing_db/billing.py:75]: I am now accessing them from the `rates` object directly. This reduces code duplication as the property wasn't doing anything besides exposing the value from the `rates` object too.

Comment (knikolla) [reply src/openstack_billing_db/billing.py:75]: I did split that out as a separate commit in this PR.

**Review by naved001 — APPROVED** [review]

**Review by joachimweyl — APPROVED** [review]
